tools/IoTEdgeDevOps/DevOpsLibTest/AgentMatrixTest.cs

```diff
@@ -182,14 +182,20 @@ public void TestProperties()
                 {
                     new AgentCapability("run-e2e-tests", "true"),
                 });
+            var connectivityTestsds = new AgentDemandSet(
```

Comment:
nit:typo

Reply:
it is not typo; it means connectivity tests demand set.  :P